test/lib/code_corps/policy/conversation_test.exs

```diff
@@ -113,4 +113,22 @@ defmodule CodeCorps.Policy.ConversationTest do
       assert show?(user, conversation)
     end
   end
+
+  describe "update?" do
+    test "returns true when user is admin" do
+      user = insert(:user, admin: true)
+      message = insert(:message)
+      conversation = insert(:conversation, message: message, user: user)
+
+      assert update?(user, conversation)
+    end
+
+    test "returns false when user is not an admin" do
+      user = insert(:user, admin: false)
+      message = insert(:message)
+      conversation = insert(:conversation, message: message, user: user)
+
+      refute update?(user, conversation)
+    end
```

Comment:
This will need tests added like in the `show?` tests above once the policy is updated per the above comment.